cover.c

```diff
@@ -174,7 +266,8 @@ static void cover_node_ensure(CoverNode* node, int line) {
     /* GLOG(("Growing map for [%s] from %d to %d - %p", node->file, node->alen, size, node->lines)); */
 
     /* realloc will grow the data and keep all current values... */
-    GMEM_REALLOC(node->lines, unsigned char*, node->alen * sizeof(unsigned char*), size * sizeof(unsigned char*));
+    GMEM_REALLOC(node->lines,  unsigned char*, node->alen * sizeof(unsigned char*), size * sizeof(unsigned char*));
+    GMEM_REALLOC(node->phases, unsigned char*, node->alen * sizeof(unsigned char*), size * sizeof(unsigned char*));
```

Comment:
My choice to use a full byte per line was born from pure ignorance and lack of skills to do it more cleverly, please feel free to do it better.